### ISSUE TITLE
Input_datetime.date value does not change from entities Card

### ISSUE THREAD

**Issue by Ives25** [home-assistant/frontend#8887]
### Checklist

I have updated to the latest available Home Assistant version.
I have cleared the cache of my browser.
I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

The change of date in the Card does not change the value of input_datetime.date (visible in State of Developper Tools)

### Describe the behavior you expected

The date selected in the Card must be assigned to input_datetime.date (as with input_datetime.heure)

### Steps to reproduce the issue

1) In HA Configuration / Helpers -> ADD HELPER -> Date and / or time -> Name: Date test -> check Date -> Create

2) In the Dashboard -> add a Card entities -> select input_Datetime.date_test

3) View the state of input_Datetime.date_test with Developer Tools / STATES

4) Change the date in the Card 

1) In Developer Tools the state of input_Datetime.date_test does not change

### What version of Home Assistant Core has the issue?

core-2021.4.0

### State of relevant entities

```yaml
# Paste your state here.

```

### Problem-relevant frontend configuration

```yaml
# Paste your YAML here.

```

### Javascript errors shown in your browser console/inspector

```txt
# Paste your logs here.

```

**Comment by bramkragten** [comment]
Update core, this was fixed in 2021.4.2

**Comment by Ives25** [comment]
I am so sorry. Thank you
